ballsdex/core/models.py

```diff
@@ -85,6 +86,7 @@ class Special(models.Model):
     democracy_card = fields.CharField(max_length=200)
     dictatorship_card = fields.CharField(max_length=200)
     union_card = fields.CharField(max_length=200)
+    emoji = fields.CharField(max_length=40)
```

Comment:
Added a description and reduced the max length (shouldn't be higher than an ID)
```suggestion
    emoji = fields.CharField(
        max_length=20,
        description="Either a unicode character or a discord emoji ID"
    )
```